### ISSUE TITLE
CR-6 SE Screen Gray on boot

### ISSUE THREAD

**Issue by osxfr33k** [CR6Community/CR-6-touchscreen#66]
I see one little green dot must be dead pixel anyhow now I see 3 files inside the kernel update folder am I suppose to put all three files into the dwin-Set folder?  I wish the instructions were clear about this.  I see one UI file and two OS files in the kernel folder.  My Motherboard version is Era v1.1.03.

**Comment by ritchiedc** [comment]
Per the instructions in the readme included in the distributed zip, "If you have never run the Community Firmware release 5 beta or higher and you have a Creality CR-6, move the files in the "CR-6 kernel upgrade" directory to the DWIN_SET directory."

**Comment by Sebazzz** [comment]
Yes, that's right. Might want to add an additional file to document this in the folder.

**Comment by osxfr33k** [comment]
OK so I never updated to the community build and going forward once I update the kernel, I won 't be doing this part again?  I see my Screen version is 4.0 so its downgraded v3.5?  Anyhow there are two OS bin files in there a v10 No RX8130 and one that is v11 RTC RX8130 and a 3rd bin "T5L_UI_DGUS2_V35_20200730".  So put all three in the "DWIN_SET" folder along with the rest of the files already in that folder?   Also is this downgrading from the current version 4.0 to 3.5? This is all confusing you have a boot loader only on the printer and a OS on the screen.  :).  It seems to have worked but I had a bit of an issue and had to flash the printer bootloader (bin file) a few times.   I kept getting a message Loading is taking longer than expected the bootloader might be confused.  Try formatting the SD card, renaming the bin file, or try a different card.   I decided to format the card again but this time format it like the SD card for the screen 4096 bytes is that ok to do that for the flashing part to the printer? It seems to finally load but if I do a factory reset I get that same message once powered off and back on it ok again.  Maybe a bug on this release with the newer CR-6 boards v1.1.01?

Edited 11-10-2021  

I put the factory firmware back and just bought another CR-6 SE and the process worked flawless with all three kernel files put into the DWIN_SET folder.  My Printer SD Card was formatted to 4096 bytes just in case and did not rename the file as I did before starting which I think was the issue.  I saw on youtube that they had a problem and renamed the file to something simple but in my case it did not like the renaming of the file so I just left the name to its original filename.
